.travis.yml

```diff
@@ -6,88 +6,6 @@ services:
 
 # ---------------------------- APT configuration -----------------------------
 
-.apt_xenial_config: &apt_xenial_config
```

Comment:
Shouldn't we just delete the entire `.travis.yml` file given that we no longer use Travis?

Reply:
I agree. We can drop this file, as we have switched to GitHub Actions, so the Travis build script will gradually get out of sync. If ever needed, it will be available in the version history anyway.